src/commands.py

```diff
@@ -260,9 +260,16 @@ def cmd_transaction(ep, c):
     else:
         outputBackChange = None
 
+    tx = list()
+    tx.append(["amount (" + get_current_block(ep)["currency"] + ")", amount / 100])
+    tx.append(["amount (DU " + get_current_block(ep)["currency"] + ")", amount / du])
```

Comment:
I did a temporary fix https://github.com/duniter/silkaj/pull/38/commits/87bd32869ee9b3c4678db1eb914e0d73a91c81cd for now.
I opened a ticket for that: https://github.com/duniter/silkaj/issues/42